src/HE.InvestmentLoans.Common/Models/App/CacheConfig.cs

```diff
@@ -0,0 +1,10 @@
+namespace HE.InvestmentLoans.Common.Models.App;
+
+public class CacheConfig : ICacheConfig
+{
+    public string RedisConnectionString { get; set; }
```

Comment:
`RedisConnectionString` -> `ConnectionString`?
```suggestion
    public string RedisConnectionString { get; set; }
```

Reply:
Or we could have a separate Config for only Redis.